Platform/Samsung/sm7125/a52q.dsc

```diff
@@ -0,0 +1,27 @@
+[Defines]
+  PLATFORM_NAME                  = a52q
+  PLATFORM_GUID                  = 54d47bf2-32f2-4532-9438-1b6971dc59c0
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sm7125/sm7125.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Samsung/sm7125/a52q.fdf.inc
+
+!include Platform/Qualcomm/sm7125/sm7125.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE
```

Comment:
You can remove `-DENABLE_LINUX_SIMPLE_MASS_STORAGE` and the mainline dtb if mass storage isn't working

Reply:
agreed